Dockerfile

```diff
@@ -25,7 +25,7 @@ ADD ./requirements.txt "${SRC_PATH}/"
 ADD ./setup.py "${SRC_PATH}/"
 RUN cd "${SRC_PATH}" && pip3 install --no-cache-dir -r "requirements.txt" && python3 setup.py develop
 
-EXPOSE 9092 8080 8778
+EXPOSE 9092 8080 8004
```

Comment:
what is exposed at `8004`?

Reply:
`8004` is the port set as `JMX_PORT` environment variable. I am not sure why if there is a requirement to expose it, but bubuku exposes this port when creating the security groups (see [code](https://github.com/zalando-nakadi/bubuku/blob/6dff16effeb75b17e6a148de6740f8960713ee7d/instance_control/aws/taupage.py#L40)). 

I'd say not to remove `8778` here as jolokia is exposed via port `8778`.

Reply:
Replaced 8004 and returned 8778 https://github.com/zalando-nakadi/bubuku/pull/147/commits/7305066a9aec613ccc5cc426706fc56c06ab64d3